ui/src/app/model/edit-table.ts

```diff
@@ -32,6 +32,14 @@ export interface IIndexData {
   spDesc: boolean | undefined | string
 }
 
+export interface ISequenceData {
+  spSeqName?: string | undefined
+  spSequenceKind?: string | undefined
+  spSkipRangeMin?: string | undefined
+  spSkipRangeMax?: string | undefined
+  spStartWithCounter?: string | undefined
```

Comment:
how is min different from startWithCounter ?

Reply:
Users can choose to set a skip range or a start with counter or both. These are properties defined by the sequence team, hence I have just provided users the option to set them